client/config-overrides.js

```diff
@@ -0,0 +1,21 @@
+const { override, fixBabelImports, addLessLoader } = require('customize-cra');
+
+module.exports = override(
+  fixBabelImports('import', {
+    libraryName: 'antd',
+    libraryDirectory: 'es',
+    style: true,
+  }),
+  addLessLoader({
+    javascriptEnabled: true,
+    modifyVars: {
+      '@primary-color': '#E8505B',
```

Comment:
@rehabas You could move these less vars into another file and import it in this file - it might make it easier to find and modify these values in the future if they were in a file named `lessVariables` or something like that, instead of here in `config-overrides` where they are a little bit hidden.

Reply:
@tomduggan85 Thank you for your efforts 😊. Can you provide me with an example or a source to do this because I tried yesterday, but I could not reach the end without errors or maybe I did not understand what you mean well.

Reply:
You're welcome!  I think it could work like this:
First, create a new file called `lessVariables.js` and put the variables there:
```
export default {
  '@primary-color': '#E8505B',
  '@btn-border-radius-base': '5px',
  '@btn-primary-shadow': '0 1px 8px #142E6E1A',
  '@btn-font-weight': '400',
  '@font-family': 'Open Sans',
  '@btn-height-base': '58px',
  '@font-size-base': '18px',
};
```
And then in this file you can import and use it:
```
import lessVariables from './lessVariables.js';
...
addLessLoader({
  javascriptEnabled: true,
  modifyVars: lessVariables
})
```